index.js

```diff
@@ -59,6 +59,7 @@ module.exports.generateLayout = generateLayout;
  */
 function generateImage(packing, callback) {
     assert(typeof packing === 'object' && typeof callback === 'function');
+    if (!packing.items.length) return callback(null, new mapnik.Image(1, 1).encodeSync('png'));
```

Comment:
since this is an async function, use `.encode` instead of `encodeSync`? And possibly just do this outside of the method and reuse an `emptyImage` value?